File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int sLen = s.length();
        
        
        if (sLen % 2 == 1){
            answer = String.valueOf(s.charAt(sLen / 2));
        }else{
            answer = String.valueOf(s.charAt(sLen / 2 - 1)) + s.charAt(sLen / 2);
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.kt
class Solution {
    fun solution(s: String): String {
        var answer = if (s.length % 2 == 0) s.substring(s.length/2-1, s.length/2+1) else s.substring(s.length/2, s.length/2+1)
        
        return answer
    }
}